.github/workflows/base-wrapper.yml

```diff
@@ -54,6 +50,10 @@ on:
         type: string
         description: 'The builder image to use instead of a release or building from source (e.g. quay.io/quarkus/ubi-quarkus-mandrel:20.3-java11)'
         default: "null"
+      build-stats-tag:
+        type: string
+        description: 'The tag to use for build stats uploads of native tests (e.g. 22.3.0-dev-jdk17-prepatch-x)'
```

Comment:
I am not sure what the tag is meant to do. Group stats from the same run? Groups stats of a "category" of runs?

I wonder if we could generate such tags by combining the repository, branch, Graal/Mandrel version, Java version and commit hash.

Reply:
Yes, group stats of "similar" runs. It's one way to select a set of metrics from the backend. Say, you wanted to compare build stats of `patch1` with unpatched in terms of build time metrics. You could trigger a run with `pre-patch1` and a run with the patch, say `patch1`, and you'd be able to get these metrics from the backend by using URL `<BASE-URL>/api/v1/image-stats/tag/pre-patch1` and `<BASE-URL>/api/v1/image-stats/tag/patch1` respectively.

Of course we can generate a name for a "default" tag, but it's still useful to be able to specify a user-defined tag as specified above.

Reply:
For the time being I'm not sure if generating the tag is very useful. The likelihood of it being empty is not that great with the updated patch. For now, I'm not sure we need to do this. I can remove the default hard-coded value and just leave it empty if we need/want.